### ISSUE TITLE
TMBadgeView position and size

### ISSUE THREAD

**Issue by rkittinger** [uias/Tabman#437]
### New Issue Checklist

- [x ] Updated Tabman to the latest version
- [ ] I took a look at the [Troubleshooting Guide](https://github.com/uias/Tabman/blob/master/Docs/Troubleshooting.md)
- [x ] I searched for [existing GitHub issues](https://github.com/uias/Tabman/issues)

### Issue Description

I would like to change the position and size of the TMBadgeView. Is this possible?

**Comment by rkittinger** [comment]
I was unable to manipulate anything other than the color and text in the bar.buttons.customize closure.

For now I will have to use a SwiftBadge or UIImageView, but then I would have to determine the position of the tab...

**Comment by msaps** [comment]
@rkittinger yeah unfortunately `TMBadgeView` is quite limited for customisation in that regard. The position and size of the badge is determined by the `TMBarButton` that contains it in `layoutBadge(_ badge: TMBadgeView, in view: UIView)`. 

You could do a custom `TMBarButton` to achieve it - I couldn't really come up with a nice API that would support everything nicely for all the possible button layouts (especially with custom types). If you have any suggestions, please go ahead! 

Sorry for the slow reply - haven't had much free time of late 👍
